src/dlgRoomProperties.cpp

```diff
@@ -0,0 +1,555 @@
+                displayStrings.append(qsl("%1 {%2:%3}")
+                    .arg(QString::number(itWeightUsed.key()))
+                    .arg(tr("count",
+                            // Intentional comment to separate arguments
+                            "This text will be part of a list of room values shown, which will name the value "
+                            "itself, followed by the counted number of rooms with that very value like: "
+                            "grey {count: 2} - So please translate like counted amount, number of, etc."))
+                    .arg(QString::number(itWeightUsed.value())));
+            }
+        }
+    }
+    return displayStrings;
+}
+
+
+void dlgRoomProperties::accept()
+{
+    QDialog::accept();
+
+    // Find name to return back
+    QString newName = lineEdit_name->text();
+    bool changeName = true;
+    if (newName == multipleValuesPlaceholder) {
+        // We don't want to change then
+        newName = QString();
+        changeName = false;
+    }
+
+    // Find color to return back
+    //   This is using mChangeRoomColor and mRoomColorNumber which are updated
+    //   immediately when the player changes colors in the dialog (as opposed
+    //   to the other room data here) - They need no further review at this time.
+
+    // Find symbol to return back
+    QString newSymbol = getNewSymbol();
+    bool changeSymbol = true;
+    QColor newSymbolColor = selectedSymbolColor;
+    bool changeSymbolColor = true;
+    if (newSymbol == multipleValuesPlaceholder) {
+        // We don't want to change then
+        changeSymbol = false;
+        changeSymbolColor = false;
+    }
+
+    // Find symbol to return back
+    int newWeight = getNewWeight();
+    bool changeWeight = true;
+    if (newWeight <= -1) {
+        // We don't want to change then
+        changeWeight = false;
+    }
+
+    // Find lock status to return back
+    Qt::CheckState newCheckState = checkBox_locked->checkState();
+    bool changeLockStatus = true;
+    bool newLockStatus;
+    if (newCheckState == Qt::PartiallyChecked) {
+        // We don't want to change then
+        changeLockStatus = false;
+    } else {
+        if (newCheckState == Qt::Checked) {
+            newLockStatus = true;
+        } else { // Qt::Unchecked
+            newLockStatus = false;
+        }
+    }
+
+    emit signal_save_symbol(
+        changeName, newName,
+        mChangeRoomColor, mRoomColorNumber,
+        changeSymbol, newSymbol,
+        changeSymbolColor, newSymbolColor,
+        changeWeight, newWeight,
+        changeLockStatus, newLockStatus,
+        mpRooms);
+}
+
+
+QString dlgRoomProperties::getNewSymbol()
+{
+    if (mpSymbols.size() <= 1) {
+        return lineEdit_roomSymbol->text();
+    }
+    QString newSymbolText = comboBox_roomSymbol->currentText();
+    // Parse the initial text before the curly braces containing count
+    QRegularExpression countStripper(qsl("^(.*) {.*}$"));
```

Comment:
Non-issue. 
1) These interpunction are deliberately non-translatable, so they can indeed be used programmatically without worry. 
2) Even when they were translatable before, you put a comment to each translator, to please keep the space and bracket structure as is, because they would be needed programmatically. 
3) In fact, no language currently touches the space or bracket characters at all, at all. 
Source: https://crowdin.com/translate/mudlet/137/en-zhtw?filter=basic&value=0#93744 etc.
![image](https://user-images.githubusercontent.com/117238/200060447-3cb74d11-09e8-42f0-8483-b2a763fb4669.png)